installing/acs-installation-platforms.adoc

```diff
@@ -1,14 +1,144 @@
 :_content-type: ASSEMBLY
 [id="acs-installation-platforms"]
-= Installation platforms and methods
+= Supported platforms and installation methods
 include::modules/common-attributes.adoc[]
 :context: acs-installation-platforms
-
 toc::[]
 
 [role="_abstract"]
-{product-title} and {product-title-managed} are supported on various platforms, such as Red Hat OpenShift and Kubernetes platforms. 
+{rh-rhacs-first} and {product-title-managed} ({product-title-managed-short}) are supported on various platforms, such as {ocp} and Kubernetes platforms. 
```

Comment:
**issue:** Preview shows the text `{rh-rhacs-first}` I think you've removed it from common attributes. Multiple instances, please fix them all.

Reply:
I have introduced these attributes via https://github.com/openshift/openshift-docs/pull/55684/files#:~:text=%3Arh%2Drhacs%2Dfirst%3A%20Red%20Hat%20Advanced%20Cluster%20Security%20for%20Kubernetes%20(RHACS), so once we merge this PR the issue should be resolved, hope that makes sense :)